pkg/reconciler/v1alpha1/serverlessservice/serverlessservice_test.go

```diff
@@ -234,6 +251,31 @@ func markNoEndpoints(sks *nv1a1.ServerlessService) {
 	sks.Status.MarkEndpointsNotReady("NoHealthyBackends")
 }
 
+type deploymentOption func(*appsv1.Deployment)
+
+func deploy(namespace, name string, opts ...deploymentOption) *appsv1.Deployment {
+	one := int32(1)
+	d := &appsv1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: namespace,
+			Name:      name,
+		},
+		Spec: appsv1.DeploymentSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"label": "value",
+				},
+			},
+			Replicas: &one,
```

Comment:
I thought you created helpers do avoid this kind of stuff? :-) (*int32)(pkg.Int(1)) or you know proto.Int32(1) ;-)